src/Form/AdminSettingsForm.php

```diff
@@ -68,7 +61,7 @@ public static function create(ContainerInterface $container) {
   /**
    * {@inheritdoc}
    */
-  public function buildForm(array $form, FormStateInterface $form_state, Request $request = NULL) {
+  public function buildForm(array $form, FormStateInterface $form_state) {
```

Comment:
I talked to Ted about the dropdown and reached to come consensus on how this can be implemented. In simple words, there will be a dropdown field of minimally a "Default" option, and rest are "EU Central 1" and "US East 1" (or whatever are available for selection). We are not able to eliminate the "Default" option conveniently, because the D8 module does not know about the "default" URL - it comes from the lift.js (search for "http://api-demo-us-east-1.decision-api.acquia.com" in the file).

The story was a 3 pointer and the team barely gave it a green light. I have already expanded it to a solid 5 because I could justify the amount of tests I could add and bugs/enhancements I could bring in for the overall quality. We are adding more to this story and I now feel it is advancing into a 8, and it is starting to drag. I really think we should merge what we have already, and gather the rest in another story.